<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multistation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1591</width>
      <height>671</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resources/themes/Dark/ui_ru_RU.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_gas">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>100 %</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_photoresistor">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>100 %</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_motion">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Нет</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>100 °C</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_humidity">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>100 %</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="details_pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="details_pushButton_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="settings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu_lang">
     <property name="title">
      <string>Язык...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/icons/icons8-language-64.png</normaloff>:/resources/icons/icons8-language-64.png</iconset>
     </property>
     <addaction name="action_lang_ru"/>
     <addaction name="action_lang_en"/>
    </widget>
    <widget class="QMenu" name="menu_units">
     <property name="title">
      <string>Единицы измерения...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/icons/icons8-substitute-64.png</normaloff>:/resources/icons/icons8-substitute-64.png</iconset>
     </property>
     <addaction name="action_units_c"/>
     <addaction name="action_units_f"/>
     <addaction name="action_units_k"/>
    </widget>
    <widget class="QMenu" name="menu_theme">
     <property name="title">
      <string>Тема...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/icons/icons8-theme-50.png</normaloff>:/resources/icons/icons8-theme-50.png</iconset>
     </property>
     <addaction name="action_theme_dark"/>
     <addaction name="action_theme_bluish"/>
     <addaction name="action_theme_cloudy"/>
     <addaction name="action_theme_burgundy"/>
     <addaction name="action_theme_sunrise"/>
     <addaction name="action_theme_rosy"/>
    </widget>
    <addaction name="menu_theme"/>
    <addaction name="menu_units"/>
    <addaction name="menu_lang"/>
    <addaction name="action_connect"/>
    <addaction name="action_limits"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="info">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_info_app"/>
   </widget>
   <addaction name="settings"/>
   <addaction name="info"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_lang_ru">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Русский язык</string>
   </property>
  </action>
  <action name="action_lang_en">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Английский язык</string>
   </property>
  </action>
  <action name="action_units_c">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Градус Цельсия - °C</string>
   </property>
  </action>
  <action name="action_units_f">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Градус Фаренгейта - °F</string>
   </property>
  </action>
  <action name="action_units_k">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Кельвины - К</string>
   </property>
  </action>
  <action name="action_info_app">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/icons8-about-50.png</normaloff>:/resources/icons/icons8-about-50.png</iconset>
   </property>
   <property name="text">
    <string>О программе...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_theme_dark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Тема Dark</string>
   </property>
  </action>
  <action name="action_theme_bluish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Тема Bluish</string>
   </property>
  </action>
  <action name="action_theme_cloudy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Тема Cloudy</string>
   </property>
  </action>
  <action name="action_theme_burgundy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Тема Burgundy</string>
   </property>
  </action>
  <action name="action_theme_sunrise">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Тема Sunrise</string>
   </property>
  </action>
  <action name="action_theme_rosy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Тема Rosy</string>
   </property>
  </action>
  <action name="action_connect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/icons8-server-50.png</normaloff>:/resources/icons/icons8-server-50.png</iconset>
   </property>
   <property name="text">
    <string>Подключение...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_limits">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/icons8-budget-limit-50.png</normaloff>:/resources/icons/icons8-budget-limit-50.png</iconset>
   </property>
   <property name="text">
    <string>Пределы срабатывания...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
